### DISCUSSION TITLE
Possible to "fake" multiple sites?

### DISCUSSION THREAD

**Discussion by ptrsmk** [plausible/analytics#714] [Self-Hosted Support]
I have a site with content from multiple authors.  I would like to be able to provide each author with their own analytics data, but not mix in data from other authors' content.  I thought that if I (1) setup a fake subdomain site [author-name.exampl.com] in Plausible and then (2) loaded plausible.js with `data-domain` equal to that fake subdomain, then Plausible would record that traffic for the fake subdomain.  However, that did not appear to work.  Is there a way to accomplish this?

**Comment by Vigasaurus** [comment]
All plausible uses to distinguish sites is the domain you provide on creation and the data-domain; if you set both correctly (to any arbitrary value) plausible will indeed send events to the appropriate dashboard. 

I'd recommend you double check that the data domain is updated and matches fully to the additional sites you made in Plausible.

**Reply by ptrsmk** [reply]
I've checked and double-checked.  Without a doubt, my data-domain string matches the domain string for the site in Plausible.  I've pulled the domain string straight from the snippet provided in the Plausible site settings area.  Plausible still says it's "Waiting for first pageview."

**Reply by metmarkosaric** [reply]
What's your domain name? Feel free to contact us via email if you prefer not to share it publicly

**Reply by ptrsmk** [reply]
@metmarkosaric 
https://peterssquare.com

Presently and for the last 45 minutes, I have the data-domain set to the fake subdomain url that I created within Plausible.

**Reply by metmarkosaric** [reply]
I just checked your site and cannot see any calls to Plausible. As you're on WordPress, I suggest you use our WordPress plugin which makes it easy to integrate with Plausible. See https://plausible.io/docs/wordpress-integration

**Reply by ptrsmk** [reply]
![image](https://user-images.githubusercontent.com/6360014/107860610-225c3d80-6e0e-11eb-950c-d5aae193ed09.png)

**Reply by ptrsmk** [reply]
As you can see in the above image, I had copied the plausible.js file to my server, rather than having it served from my Plausible server.  I just changed it a minute ago, now having it served from the Plausible server, and it is working as expected.  The subdomain site now shows traffic.

Is there a way around this?  Some way that I can serve the plausible.js file from my site's server or from my CDN?  Is it an absolute requirement that it be served from my Plausible server?

**Reply by metmarkosaric** [reply]
You can try use our custom domain functionality https://plausible.io/docs/custom-domain

**Reply by ptrsmk** [reply]
But I am self-hosting Plausible.

**Reply by ukutaht** [reply]
> Is there a way around this? Some way that I can serve the plausible.js file from my site's server or from my CDN? Is it an absolute requirement that it be served from my Plausible server?

It should be possible. If you can afford to lose some stats you can put the script back on your own server and send me a link so I can debug and see why it's not getting recorded right.

You should see a call to `POST https://your-plausible.com/api/event` with a 202 response code.

Even without debugging I have one theory. Where did you copy the JS from? It has to be copied from your own server because the JS snippet includes the `BASE_DOMAIN` you have configured. If you copy the JS from https://plausible.io/js/plausible.js then the stats will be sent to our cloud server.
